plugins/modules/modprobe.py

```diff
@@ -35,6 +35,21 @@
         description:
             - Modules parameters.
         default: ''
+    persistent:
+        type: str
+        choices: [ disabled, absent, present ]
+        default: disabled
+        description:
+            - Persistency between reboots for configured module.
+            - This option creates files in C(/etc/modules-load.d/) and C(/etc/modprobe.d/) that make your module configuration persistent during reboots.
+            - If C(present), module adds module name to C(/etc/modules-load.d/) and params to C(/etc/modprobe.d/) so module will be loaded on next reboot.
```

Comment:
Here and below, `module` is used for two different things - the Ansible module, and the kernel module. Let's only use `module` for kernel module.
```suggestion
            - If C(present), adds module name to C(/etc/modules-load.d/) and params to C(/etc/modprobe.d/) so the module will be loaded on next reboot.
```